rules/qc/repeat_masking.rules

```diff
@@ -26,21 +26,32 @@ rule repeatmasker:
     threads:
         Cfg['qc']['threads']
     params:
-        dir = str(QC_FP/'masked'/'log')
+        dir = str(QC_FP/'masked'/'log'),
+        skip = Cfg['qc']['mask_reads'] if 'mask_reads' in Cfg['qc'].keys() else False
```

Comment:
I think this means `mask_reads` actually behaves more like `no_mask_reads`, since that boolean gets assigned as-is to `skip` here.  So for example if I have this in my config:

    qc:
      mask_reads: True

...it'll actually skip masking.  Should we swap the config option name?  Or maybe negate the assignment here?